### PR TITLE
Fix for PCM-542,543,544,545 and also PCM-378

### PR DESCRIPTION
**Pull request by ardasgup**
@sshumake @renujhamtani  Please review and merge. Have clubbed all the escalation related JIRAs because that made sense and is independent of any other module so should not be difficult to segregate.

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -202,8 +202,9 @@
     <script src="search/services/searchResultsService.js"></script>
 
     <script src="escalation/escalation.module.js"></script>
-    <script src="escalation/controllers/partnerEscalation.js"></script>
+    <script src="escalation/controllers/escalationRequest.js"></script>
     <script src="escalation/services/escalationRequestService.js"></script>
+    <script src="escalation/directives/403.js"></script>
```
Comment (renujhamtani) [inline app/index.html]: @ardasgup  I think the new entries in index.html should also go under chromed.html. Correct me if I am wrong

**Comment by renujhamtani** [comment]
@ardasgup Reviewed except for the above comment for chromed.html.

File: app/index.html (hunk 1 above)
Comment (ardasgup) [inline app/index.html]: @renujhamtani  absolutely i forgot :( have pushed the same.

File: app/escalation/views/iceEscalation.html [hunk 2]
```diff
@@ -0,0 +1 @@
...he escalation, the business context, the competitive situation, past support experience'|translate}}</span></p><p><span>{{'If you are requesting an account review, if this is related to aspecific problem/ticket or a series of problems/tickets or if this is a general or generic complain'|translate}}</span></p><p><span>{{'If this is a request for information from you or the customer and if you need us to contact you to discuss the issues.'|translate}}</span></p></span></div></div><div class="form-group"><label for="rha-expectations" class="col-sm-5 control-label">{{'Expectation'|translate}}</label><div class="col-sm-7"><textarea id="rha-expectations" ng-model="EscalationRequestService.expectations" ng-change="iceMandatoryFieldCheck()" class="form-control nonEmpty"></textarea><span class="help-block">{{'Please provide expectations and desired outcomes from yourself and from the customer'|translate}}</span></div></div></fieldset></div><div class="clear"></div><div class="col-md-7 col-md-offset-1 push-bottom"><button ng-click="submitEscalationRequest(ESCALATION_TYPE.ice)" ng-disabled="disableSendRequest" translate="" class="btn btn-primary">Send Request</button></div></div></form></div>
```
Comment (sshumake) [inline app/escalation/views/iceEscalation.html]: html files should not be checked in

File: app/escalation/views/partnerEscalation.html [hunk 3]
```diff
@@ -0,0 +1 @@
..." class="col-sm-5 control-label">{{'Issue Description'|translate}} *</label><div class="col-sm-7"><textarea id="rha-issue-description" ng-model="EscalationRequestService.issueDescription" ng-change="partnerMandatoryFieldCheck()" class="form-control nonEmpty"></textarea><span class="help-block">{{'Please provide as much detail as possible to help us understand the issue, for example:Reason for the escalation, the business context, the competitive situation, past support experience'|translate}}</span></div></div><div class="form-group"><label for="rha-expectations" class="col-sm-5 control-label">{{'Expectation'|translate}}</label><div class="col-sm-7"><textarea id="rha-expectations" ng-model="EscalationRequestService.expectations" class="form-control nonEmpty"></textarea><span class="help-block">{{'Please provide expectations and desired outcomes from yourself and from the customer'|translate}}</span></div></div></fieldset></div><div class="clear"></div><div class="col-md-7 col-md-offset-1 push-bottom"><button ng-click="submitEscalationRequest(ESCALATION_TYPE.partner)" ng-disabled="disableSendRequest" translate="" class="btn btn-primary">Send Request</button></div></div></form></div>
```
Comment (sshumake) [inline app/escalation/views/partnerEscalation.html]: html files should not be checked in

File: app/escalation/directives/403.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+'use strict';
```
Comment (sshumake) [inline app/escalation/directives/403.js]: In the beta i have moved this entire directive into common. This is OK for now but when this is merged with BETA we should use directive in common.

Comment (ardasgup) [inline app/escalation/directives/403.js]: Ack

File: app/escalation/views/iceEscalation.html (hunk 2 above)
Comment (ardasgup) [inline app/escalation/views/iceEscalation.html]: Wrongly used git add on folder during commit which led to this, will remove.

**Comment by ardasgup** [comment]
@renujhamtani can u pls merge this

**Comment by renujhamtani** [comment]
@ardasgup Merged,

**Pull request merged**